tests/unit/core/i18n/test_utils.js

```diff
@@ -509,4 +510,73 @@ describe(__filename, () => {
       expect(i18n.moment('1988-09-22').format('ll')).toEqual('22 sept. 1988');
     });
   });
+
+  describe('formatFilesize', () => {
+    const _formatFilesize = ({
+      _filesize,
+      _log,
+      i18n = fakeI18n(),
+      size = 123,
+    }) => {
+      return utils.formatFilesize({ _filesize, _log, i18n, size });
+    };
+
+    it('receives an expected string from filesize', () => {
+      // This is a sanity check, verifying that filesize does in fact return
+      // what we expect.
+      // Without it filesize could break and we'd never know.
```

Comment:
I like this test but do you really need to use a mock version of `filesize()` for the other tests? What about using the real function?

Reply:
I just figured that mocking was better, so that we're isolating the test to the code we're concerned with. I guess I can use the real function everywhere, in which case I probably don't even need to configure the function to accept a mock. That seems contrary to the way we usually write tests though.